MVP-java/app/src/main/java/com/base/mvp/core/di/module/ApplicationModule.java

```diff
@@ -43,28 +48,46 @@ Application provideApplication() {
         return application;
     }
 
+    @Provides
+    @DatabaseInfo
+    String provideDatabaseName() {
+        return AppConstants.DB_NAME;
```

Comment:
DB_NAME: i think you should change name

Reply:
yes, i will change now and will create new pull request